src/main/kotlin/io/perfometer/dsl/HttpDsl.kt

```diff
@@ -63,13 +71,13 @@ class HttpDsl(
         private val host: String,
         private val port: Int,
 ) {
-    private val steps: MutableList<Step> = mutableListOf()
+    val steps: MutableList<Step> = mutableListOf()
```

Comment:
Is it necessary to expose the steps list as Mutable interface?

EDIT: From what I see below, it's not - please change the interface to `List` so that it appear as immutable for the external world, or consider leaving this field as private, and adding a immutable "view" of the property:

```
val scenarioSteps: List<Step>
  get() = this.steps
```

Reply:
done